test/unit-test/test_restore.c

```diff
@@ -388,21 +389,21 @@ int main(void)
 {
     const struct CMUnitTest tests[] = {
         cmocka_unit_test_setup_teardown(test_restore_from_buffer, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_restore_list_backups_single_seed, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_restore_list_backups_multiple_seeds, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_restore_good_backup_from_directory, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_restore_1_corrupt_backup_from_directory, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_restore_2_corrupt_backups_from_directory, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_restore_3_corrupt_backups_from_directory, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_fail_restore_if_majority_vote_fails, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(test_restore_seed, test_setup, test_teardown),
+        /* cmocka_unit_test_setup_teardown( */
```

Comment:
Wait, why are these commented out now, all except the first test?

Reply:
oh that's embarrassing :O fixed

Reply:
Ah cool. For a second I thought they somehow weren't truly working anymore. :)